Gemfile

```diff
@@ -107,6 +107,7 @@ group :development, :test do
   gem "erb_lint", "0.9.0", require: false
   gem "hirb"
   gem "htmlentities"
+  gem "httpclient"
```

Comment:
I can see this is needed, but it's for a single test, it might be worth dropping a tech-debt ticket to update the test and remove the Gem again! 😃

Reply:
have removed the httpclient gem now, and modified the tests